READMEdone.md

```diff
@@ -0,0 +1,35 @@
+# Setup 
+To setup database - there is the psql.yml file for docker-compose within psql.env for environement variables
+
+All necessary libraries are in 'requirements.txt'
```

Comment:
There is no such file provided in repository

Reply:
It is ‘psql.yml’

Reply:
In the root of project

Reply:
I mean `requirements.txt`

Reply:
I see now, accidentally miss it. Added

Reply:
Tnx